libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -4,6 +4,8 @@
       autoFocus
       matInput
       type="search"
+      [(ngModel)]="searchData"
+      (keyup)="searchBooksReason()"
```

Comment:
Since a reactive form is used in this component, this hints that it would be preferable to use reactive form methods to handle value changes.

Also, you cannot use both reactive forms (`formControl`) and `ngModel` in the same form field. If you run the tests you will see this warning:
```
          It looks like you're using ngModel on the same form field as formControlName.
          Support for using the ngModel input property and ngModelChange event with
          reactive form directives has been deprecated in Angular v6 and will be removed
          in a future version of Angular.
```

Please take extra care with the forms! :)